### PR TITLE
New "valueFormat" prop

### PR DESCRIPTION
**Pull request by corpglory-dev**
Now progress-list can only display percentage values for progress bars.
This PR adds optional `valueFormat` prop to be able to display absolute values in progress bars.

## Changes:
 - new `valueFormat` prop with enum type: `{ ABSOLUTE = "absolute", PERCENTAGE = "percentage" }` and `"percentage"` default value
 - move `maxValue` and `opacity` fields out of the `config` prop
 - new `stops: { values: StopValues, colors: StopColors }` prop
 - fix docs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,18 +26,47 @@ Result:
...
   { title: 'Bar', value: 100, backgroundColor: 'red' }
 ]
 ```
-- `config` — progress-list config, e.g:
+- `config` — (required) progress-list config, e.g:
 ```
 :config={ maxValue: 100 }
 ```
+- `maxValue` - (optional) maximum of item`s value, e.g:
+```
```
Comment (jonyrock) [inline README.md]:
```suggestion
```js
```

File: README.md [hunk 2]
```diff
@@ -26,18 +26,47 @@ Result:
 ![image](https://user-images.githubusercontent.com/47055832/70167248-5a7ad300-16d7-11ea-96ac-f5311dc58b59.png)
 
 ## Props
-- `items` — array of configs for each progress bar, e.g: 
+- `items` — (required) array of configs for each progress bar, e.g:
 ```
```
Comment (jonyrock) [inline README.md]:
```js

File: README.md [hunk 3]
```diff
@@ -26,18 +26,47 @@ Result:
...
+- `maxValue` - (optional) maximum of item`s value, e.g:
+```
+:maxValue=100
+```
+- `opacity` - (optional) css stype for progress bars, e.g:
+```
+:opacity=0.5
+```
+- `threshold` - (optional) config with threshold values and colors, eg:
+```
```
Comment (jonyrock) [inline README.md]:
```js

File: README.md [hunk 4]
```diff
@@ -26,18 +26,47 @@ Result:
...
+    low: 'green'
+  }
+}
+```
+Result:
+
+![image](https://user-images.githubusercontent.com/39257464/70325419-fb39d180-1842-11ea-832c-01b25be6c3fa.png)
 
+- `valueFormat` - (optional) format of item`s values, 'persentage' or 'absolute', eg:
+```
```
Comment (jonyrock) [inline README.md]:
```js

File: src/components/ProgressItem.vue [hunk 5]
```diff
@@ -4,24 +4,30 @@
     <div class="list-item-content">
       <div
         v-bind:style="{
-          width: progress + '%',
+          width: percentageProgress,
```
Comment (jonyrock) [inline src/components/ProgressItem.vue]:
```suggestion
          width: barWidth,
```

File: src/components/ProgressItem.vue [hunk 6]
```diff
@@ -4,24 +4,30 @@
...
-import { Item, Config, Thresholds, ThresholdConfig } from './types';
+import { ValueFormat, Item, Config, Threshold, Thresholds, ThresholdColors } from './types';
 
 import { Component, Vue, Prop, Watch } from 'vue-property-decorator';
 
+const DEFAULT_THRESHOLD_COLORS = {
+  low: 'green',
+  medium: 'orange',
+  high: 'red'
+}
```
Comment (jonyrock) [inline src/components/ProgressItem.vue]:
```suggestion
};
```

File: src/components/ProgressItem.vue [hunk 7]
```diff
@@ -31,26 +37,67 @@ export default class ProgressItem extends Vue {
...
+  valueFormat!: ValueFormat;
+
+  get progress(): string {
+    if(this.valueFormat === ValueFormat.PERCENTAGE) {
+      return this.percentageProgress;
+    }
+    return this.item.value.toString();
+  }
+
+  get percentageProgress(): string {
```
Comment (rozetko) [inline src/components/ProgressItem.vue]:
```suggestion
  get width(): string {
```

**Review by jonyrock — CHANGES_REQUESTED** [review]
small styles fixes

File: README.md [hunk 8]
```diff
@@ -26,18 +26,47 @@ Result:
...
+  }
+}
+```
+Result:
 
+![image](https://user-images.githubusercontent.com/39257464/70325419-fb39d180-1842-11ea-832c-01b25be6c3fa.png)
+
+- `valueFormat` - (optional) format of item`s values, 'persentage' or 'absolute', eg:
+```js
+:valueFormat='abosule'
```
Comment (corpglory-dev) [inline README.md]:
```suggestion
:valueFormat='abosute'
```

**Pull request merged**